tests/conftest.py

```
@@ -218,3 +220,39 @@ def jwt_grpc_tenant2(fulfillment_address: str, keycloak_url: str, jwt_password:
         keycloak_url=keycloak_url, realm="osac", client_id="osac-cli", username="tenant2_user", password=jwt_password
     )
     return GRPCClient(address=fulfillment_address, token=token)
+
+
+# --- Cross-cutting concern: Metering ---
+
+
+def pytest_collection_modifyitems(config: pytest.Config, items: list[pytest.Item]) -> None:
```

Ruff (0.16.0)

[warning] Unused function argument: `config`

(ARG001)